app/screens/create_or_edit_channel/channel_info_form.tsx

```diff
@@ -0,0 +1,345 @@
+                                    disableFullscreenUI={true}
+                                    enablesReturnKeyAutomatically={true}
+                                    label={labelPurpose}
+                                    placeholder={placeholderPurpose}
+                                    onChangeText={onPurposeChange}
+                                    keyboardAppearance={getKeyboardAppearanceFromTheme(theme)}
+                                    returnKeyType='next'
+                                    showErrorIcon={false}
+                                    spellCheck={false}
+                                    testID='edit_channel_info.purpose.input'
+                                    value={purpose}
+                                    ref={purposeInput}
+                                    containerStyle={styles.textInput}
+                                    theme={theme}
+                                />
+                                <FormattedText
+                                    style={styles.helpText}
+                                    id='channel_modal.descriptionHelp'
+                                    defaultMessage='Describe how this channel should be used.'
+                                />
+                            </>
+                        )}
+                        <FloatingTextInput
+                            autoCorrect={false}
+                            autoCapitalize={'none'}
+                            blurOnSubmit={false}
+                            disableFullscreenUI={true}
+                            enablesReturnKeyAutomatically={true}
+                            label={labelHeader}
+                            placeholder={placeholderHeader}
+                            onChangeText={onHeaderChange}
+                            multiline={true}
+                            keyboardAppearance={getKeyboardAppearanceFromTheme(theme)}
+                            returnKeyType='next'
+                            showErrorIcon={false}
+                            spellCheck={false}
+                            testID='edit_channel_info.header.input'
+                            value={header}
+                            onLayout={onHeaderLayout}
+                            ref={headerInput}
+                            containerStyle={styles.textInput}
+                            theme={theme}
+                        />
+                        <FormattedText
+                            style={styles.helpText}
+                            id='channel_modal.headerHelp'
+                            defaultMessage={'Specify text to appear in the channel header beside the channel name. For example, include frequently used links by typing link text [Link Title](http://example.com).'}
+                        />
+                    </View>
+                </TouchableWithoutFeedback>
+
+            </KeyboardAwareScrollView>
+            <View>
```

Comment:
Do you need the <View> component?

Reply:
Which one? The ScrollView? Why do you think we may not need it?

Reply:
🤷‍♂️  this looks fine to me

Reply:
I mean the wrapping `<View>` around the `<Autocomplete>`

Reply:
Because Autocomplete is absolute this view makes it relative to it